src/redux/actions/projects/deleteProject.js

```diff
@@ -47,6 +47,14 @@ const deleteProject = (
       throw new Error(response.json().message);
     }
 
+    // If deleted project is the same as the active project, choose another project
+    if (projectUuid === activeProjectUuid) {
+      dispatch({
+        type: PROJECTS_SET_ACTIVE,
+        payload: { projectUuid: projects.ids.length > 1 ? projects.ids[0] : null },
```

Comment:
Do we have any guarantee that `projects.ids[0] != projectUuid` and that we are not going to try to set as the active project the one that we are deleting?

Reply:
Good point